### PR TITLE
add Rss feed to events

### LINKED ISSUE #2554: [Events] RSS Feed: Create feed for all posts
**Current Behavior**
Site visitors can browse [events](https://layer5.io/community/events), but not subscribe for notifications of upcoming events.

**Desired Behavior**
Create `/events/feed.xml` - an RSS feed containing all posts.

---
**Screenshots / Mockups**
Screenshot 2022-01-29 at 1 19 23 AM

**Implementation**
1. Create new RSS feed in `gatsby-config.js` to be available at `/events/feed.xml`.
1. Add RSS icon with link to feed adjacent to /events page title as pictured in mockup. Check [blog page](https://layer5.io/blog) for example.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Sunnyio**
Signed-off-by: Sunny Kumar <sunny514173@gmail.com>

**Description**

This PR fixes #2554

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 54f7806805c192eee52e8ec5188588d17be7b68f at: https://61f50915d5bc6ee5fed06078--layer5ng.netlify.app

File: gatsby-config.js [hunk 1]
```diff
@@ -241,6 +241,53 @@ module.exports = {
...
+                    fields {
+                      collection
+                      slug
+                    }
+                  }
+                }
+              }
+            `,
+            output: "/events/feed.xml",
+            title: "Layer5"
```
Comment (Nikhil-Ladha) [inline gatsby-config.js:289]: Make it `Layer5 Events`.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by Nikhil-Ladha** [comment]
Also, @Sunnyio once this PR is merged please fetch the latest code from upstream and update your fork. I see the file contains some old data, which might create conflicts in future PRs.

**Comment by Sunnyio** [comment]
Sure @Nikhil-Ladha. Can you please review it again?

**Comment by l5io** [comment]
🚀 Preview for commit 1ad32ca976c5b55b2a8597969cca4eb2fea6f8bd at: https://61fb74b116218c265a7c5bb7--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**